teflo_datarouter_plugin/datarouter_plugin.py

```diff
@@ -44,8 +43,10 @@ class DataRouterPlugin(ImporterPlugin):
     __schema_ext_path__ = os.path.abspath(os.path.join(os.path.dirname(__file__),
                                                        "files/schema_extensions.py"))
 
-    def __init__(self, report):
+    def __init__(self, report, dr_client_secret=None, dr_client_id=None):
```

Comment:
remove dr_client_Secret and id from signature and declare them as none in init method